### PR TITLE
Updated import from PDC Describe to use the new structure

### LINKED ISSUE #373: Import proper Collection value form PDC Records
We are currently not exporting the collection on PDC Records (see https://github.com/pulibrary/pdc_describe/issues/841). Once that has been fixed we should update the import process to pick up the value. 

In PR https://github.com/pulibrary/pdc_discovery/pull/372 I hard-coded the collection to "Research Data" but that value can be updated once this issue is addressed.

### PR DESCRIPTION
**Pull request by hectorcorrea**
Now that PDC Describe includes resource + collection + file information I refactored the ingestion process to use the new format. The main change for this is using the proper path to the properties in `./lib/traject/pdc_describe_indexing_config.rb`

I also took the time to use the new property for the Collection rather than using a hard-coded value (fixes #373). 

However, during this process I noticed that we had a bunch of duplicate tests in `./spec/lib/describe_indexer_spec.rb` and `./spec/lib/pdc_describe_indexer_spec.rb`. How in the world did I not see that before it's beyond me (particularly since I added `describe_indexer_spec` just a few days ago) -- but this time I took the time to remove the duplicates.

### CODE REVIEW

File: lib/traject/pdc_describe_indexing_config.rb [hunk 1]
```diff
@@ -21,87 +21,71 @@
...
-end
-
-to_field 'community_path_name_ssi' do |_record, accumulator, _c|
-  # TODO: pick the correct field from the PDC Describe source once it is available
-  # see https://github.com/pulibrary/pdc_describe/issues/841
-  accumulator.concat ["Research Data"]
-end
+to_field 'community_name_ssi', extract_xpath("/hash/collection/title")
+to_field 'community_root_name_ssi', extract_xpath("/hash/collection/title")
+to_field 'community_path_name_ssi', extract_xpath("/hash/collection/title")
```
Comment (hectorcorrea) [inline lib/traject/pdc_describe_indexing_config.rb:47]: We now get this value from PDC Describe wo we can import it.

File: spec/fixtures/files/bitklavier_binaural.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (hectorcorrea) [inline spec/fixtures/files/bitklavier_binaural.json:1]: Notice that the new fixtures have a `resource` property but also `collection` and `files`, hence the big refactor.

File: spec/lib/describe_indexer_spec.rb [hunk 3]
```diff
@@ -81,5 +93,60 @@
...
+
+    context "files" do
+      # TODO: Index filenames for PDC Describe objects
+      xit 'files' do
+        # The fixture has three files but we expect two of them to be ignored
+        files = JSON.parse(result['files_ss'].first)
+        expect(files.count).to eq 1
+        expect(files.first["name"]).to eq 'Lee_princeton_0181D_10086.pdf'
+      end
+    end
```
Comment (hectorcorrea) [inline spec/lib/describe_indexer_spec.rb:105]: This came from the duplicate (and now deleted) file `spec/lib/pdc_describe_indexer_spec.rb`

File: spec/lib/describe_indexer_spec.rb [hunk 4]
```diff
@@ -81,5 +93,60 @@
...
+  end
+
+  describe 'indexing an RSS feed from PDC Describe' do
```
Comment (hectorcorrea) [inline spec/lib/describe_indexer_spec.rb:108]: This came from the duplicate (and now deleted) file `spec/lib/pdc_describe_indexer_spec.rb`

File: spec/system/bitklavier_single_item_metadata_spec.rb [hunk 5]
```diff
@@ -31,7 +31,7 @@
   it "has expected metadata" do
     visit '/catalog/doi-10-34770-r75s-9j74'
     expect(page).to have_content "bitKlavier Grand Sample Library—Binaural Mic Image"
-    authors = "<span>Trueman, Daniel; Wang, Matthew; Villalta, Andrés; Chou, Katie; Ayres, Christien</span>"
+    authors = "<span>Trueman, Daniel</span>"
```
Comment (hectorcorrea) [inline spec/system/bitklavier_single_item_metadata_spec.rb:34]: The new fixture has only one author (the rest are now contributors and that's tested on the indexing spect)

**Review by bess — APPROVED** [review]
🚀

**Pull request merged**